### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by cchayden**
Updated documentation after going through a fresh install.

### CODE REVIEW

File: templates/jupyterhub-instructions.md [hunk 1]
```diff
@@ -315,44 +375,48 @@ Let's build and install the docker.
...
 
     http://jenkins.life.atigeo.com/job/xpatterns-token-service-docker/
 
 This does not trigger on checkin.
+Click on "Build With Parameters" to build it.
 
 Configure jupyterhub to use the token service
 ---------------------------------------------
-Go to julyterhub and add
+Go to julyterhub_config.py and add
```
Comment (kebwi) [inline templates/jupyterhub-instructions.md]: Should this really be "julyterhub_config.py"?  Or "jupyterhub_config.py" instead?  Looks like a typo to me.

**Comment by kebwi** [comment]
I added a comment to the code.  Can you see it?

**Comment by kebwi** [comment]
Ship it, conditional on your own assessment of the comment apropos to the typo.

File: templates/jupyterhub-instructions.md (hunk 1 above)
Comment (cchayden) [reply templates/jupyterhub-instructions.md]: Typo fixed.

**Pull request merged**